docs/en/observability/synthetics-params-secrets.asciidoc

```diff
@@ -78,17 +73,74 @@ The example above uses the `env` variable, which corresponds to the value of the
 
 [discrete]
 [[synthetics-cli-params]]
-= Use CLI arguments to set params
+== CLI argument
 
 To set parameters when running `elastic-synthetics` on the command line, use the `--params` or `-p` flag to the `elastic-synthetics` program. The provided map is merged over any existing variables defined in the `synthetics.config.{js,ts}` file.
 
-For example, to override the `url` parameter, you would run:
+For example, to override the `my_url` parameter, you would run:
 
 [source,sh]
 ----
-elastic-synthetics . --params '{"url": "http://localhost:8080"}'
+elastic-synthetics . --params '{"my_url": "http://localhost:8080"}'
+----
```

Comment:
@paulb-elastic @dominiqueclarke sorry for the late addition, but I'm just realizing that this example should probably be updated from `elastic-synthetics` to `npx @elastic/synthetics`, too, right? Like:

>To set parameters when running [`npx @elastic/synthetics` on the command line](https://observability-docs_2922.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-command-reference.html), use the `--params` or `-p` flag. The provided map is merged over any existing variables defined in the `synthetics.config.{js,ts}` file.
>
>For example, to override the `my_url` parameter, you would run:
>
>```sh
>npx @elastic/synthetics . --params '{"my_url": "http://localhost:8080"}'
>```

Reply:
@colleenmcginnis yes, good catch - the current version is only partially there

Reply:
Ok! I'll make this change then merge. If @dominiqueclarke has any suggestions I can open a follow-up PR!